### PR TITLE
scripts: Remove invalid warning message in iso-manager script

### PR DESCRIPTION
**Pull request by TeddyAndrieux**

### CODE REVIEW

File: scripts/iso-manager.sh [hunk 1]
```diff
@@ -28,11 +28,15 @@ ARCHIVE_DEPRECATED_OPTION=
 
 while (( "$#" )); do
   case "$1" in
-    -a|--archive|--add-archive)
+    --archive)
       ARCHIVE_DEPRECATED_OPTION="true"
```
Comment (gdemonet) [inline scripts/iso-manager.sh]: Or maybe just

```suggestion
      [ "$1" = "--archive" ] && ARCHIVE_DEPRECATED_OPTION="true"
```

?

Comment (TeddyAndrieux) [reply scripts/iso-manager.sh]: Yes, ok ok

Comment (TeddyAndrieux) [reply scripts/iso-manager.sh]: Fixed

**Review by gdemonet — APPROVED** [review]

**Pull request merged**